### ISSUE TITLE
The script is failing to work in non english installs

### ISSUE THREAD

**Issue by aseques** [zenoss/microsoft.tools#18]
I have a windows install set in spanish, and one of the issues I had is that the scripts to restrict the users to the proper permissions do not work. It basically fails because the groups have different names depending on the language, and the variable $localgroups not always the same

Original content

```
$localgroups = @(
    "Performance Monitor Users",
    "Performance Log Users", 
    "Event Log Readers", 
    "Distributed COM Users", 
    "WinRMRemoteWMIUsers__"
    )
```

Group names in spanish

```
$localgroups = @(
    "Usuarios del monitor de sistema",
    "Usuarios del registro de rendimiento", 
    "Lectores del registro de eventos", 
    "Usuarios COM distribuidos", 
    "WinRMRemoteWMIUsers__"
    )
```

The correspondency would be like this:
- Event Log Readers -> Lectores del registro de eventos
- Distributed COM Users -> Usuarios COM distribuidos
- Performance Monitor Users -> Usuarios del monitor de sistema
- Performance Log Users -> Usuarios del registro de rendimiento

This is what I get when adding the privileges to an user called monitloc

```
Registry Security Descriptor failed for HKLM:\software\microsoft\ole
Group does not exist: Performance Monitor Users
Group does not exist: Performance Monitor Users
Group does not exist: Performance Log Users
Group does not exist: Performance Log Users
Group does not exist: Event Log Readers
Group does not exist: Event Log Readers
Group does not exist: Distributed COM Users
Group does not exist: Distributed COM Users
Group does not exist: WinRMRemoteWMIUsers__
Group does not exist: WinRMRemoteWMIUsers__
Zenoss Resource Manager security permissions have been set for SATAPPSDEV\monitorloc
```

I couldn't find a precreated group matching WinRMRemoteWMIUsers__,  I can only connect properly if the user is also a member from the Administrator group (that is what we wanted to avoid in first term)
NOTE: I changed the value of the ExecutionPolicy directive to both unrestricted or remotesigned without any difference

**Comment by aseques** [comment]
In case someone else is looking into this, applying the english LIP for the system doesn't translate the group names, but still, if the permissions are ok, most of the microsoft template will work

**Comment by aseques** [comment]
This issue was fixed with commit c8037bb2b80c9948be1c2ca4bf26755b31a0efb2, using the SID is language independent. I'm closing this issue
